tests/test_cli.py

```diff
@@ -20,6 +25,51 @@ def context():
     return ctx().obj["context"]
 
 
+mock_gui_main = """from .app import show
```

Comment:
I had a go with writing these tests. I'm not sure what the previous tests were doing, the Mock GUI was importing Click? That's some inception stuff going on there.

Have a look at what you think of the current tests.